modules/rostests/kmtests/ntos_cc/CcCopyRead_drv.c

```diff
@@ -143,6 +141,7 @@ MapAndLockUserBuffer(
             _SEH2_YIELD(return NULL);
         }
         _SEH2_END;
+// Q: Where will MmUnlockPages() and IoFreeMdl() happen? Should add them to IRP_MJ_CLEANUP case? // Fix broken code from r72165?
```

Comment:
MdlAddress will never be NULL in this test case. Cc allocates it before the call.

Reply:
Ah, thanks for the confirmation. Then, this is dead code. Replaced with an `ASSERT`.